software/Test_Robot_Software/api.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+import sys
+import time
+import struct
+
+import serial
+s = serial.Serial("/dev/ttyACM0", baudrate=115200)
+
+
+def sendThrower(thrower_speed):    
+    pass
+
+def sendMotorSpeed(motors):
+    print(motors)
+    data = struct.pack('<hhhHBH', motors[0], motors[2], motors[1], 48, 0, 0xAAAA)
+    s.write(data)
+    s.read()    
+    
+def close():
+    s.close() 
+
+
+def move(x_speed, y_speed, rot_speed):
+    
+    disable_failsafe = 1
+    speeds = [0, 0, 0]
+    robotSpeed = math.sqrt(x_speed * x_speed + y_speed * y_speed)
+    robotDirectionAngle = math.atan2(y_speed, x_speed)
+
+    for wheelAngle, i in zip([240, 120, 0], range(3)):
+        wheelLinearVelocity = robotSpeed * math.cos(robotDirectionAngle - wheelAngle / 180.0 * math.pi) + rot_speed
+        speeds[i] = wheelLinearVelocity
+
+    print(x_speed, y_speed, rot_speed)
+    print(list(map(lambda x: int(x), speeds)))
```

Comment:
Why do these conversions here? For print?

Reply:
Yes, it prints here and sends to the motor driving function